04/1_string_case_insensitive_compare/main.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+
+#include "string.h"

```

Comment:
c already has string.h system library so let's call it something different

string_utils.h ?